gridstatus/ieso.py

```diff
@@ -844,3 +910,788 @@ def _request(self, url, verbose):
             )
 
         return r
+
+    @support_date_range(frequency="DAY_START")
+    def get_resource_adequacy_report(
+        self,
+        date: str | datetime.date | datetime.datetime,
+        end: datetime.date | datetime.datetime | None = None,
+        vintages: Literal["all", "latest"] = "latest",
```

Comment:
In `ISONEAPI` methods, we use singular "vintage" for the parameter. Should we be consistent?

Reply:
done.